fighthealthinsurance/models.py

```
@@ -635,15 +632,296 @@ def _get_filename(self) -> str:
+            file = file_field.open()
+            encrypted_bytes = file.read()
+            if not encrypted_bytes:
+                return None
+
+            # Try to decrypt (EncryptedFileField stores encrypted data)
+            try:
+                decrypted_bytes = Cryptographer.decrypted(encrypted_bytes)
+                logger.debug(
+                    f"Successfully read and decrypted {field_name} for fax {self.fax_id}"
+                )
+                return decrypted_bytes
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)